Tests/Unit/NetworkServer/MessageProcessorJoinTest.cs

```diff
@@ -58,14 +55,15 @@ public async Task When_Device_Is_Found_In_Api_Should_Update_Twin_And_Return()
             var simulatedDevice = new SimulatedDevice(TestDeviceInfo.CreateOTAADevice(1, gatewayID: ServerConfiguration.GatewayID));
             simulatedDevice.LoRaDevice.NwkSKey = string.Empty;
             simulatedDevice.LoRaDevice.AppSKey = string.Empty;
+
             var joinRequest = simulatedDevice.CreateJoinRequest();
-            joinRequest.DevNonce = new DevNonce(0x9786);
 
-            // Create Rxpk
-            var rxpk = joinRequest.SerializeUplink(simulatedDevice.LoRaDevice.AppKey).Rxpk[0];
+            // this former join request is just to set the loradevice cache to another devnonce.
+            var formerjoinRequest = simulatedDevice.CreateJoinRequest();
```

Comment:
Do we need to assign the result to a variable if it's just to set the cache?

Reply:
You are right ! Fixed